catalog.html

```diff
@@ -218,9 +219,11 @@ <h2 class="distribution-title">Подпишитесь на рассылку</h2>
           <button class="button-blue" type="submit">Подписаться</button>
         </form>
       </section>
+      </div>
     </main>
 
     <footer class="page-footer">
+      <div class="page-footer-container">
       <section class="footer-social">
```

Comment:
```suggestion
      <div class="page-footer-container">
        <section class="footer-social">
```
так же, выровняй вложенность